src/app/routes/utils/fetchPageData/index.test.js

```diff
@@ -157,6 +159,26 @@ describe('fetchPageData', () => {
           error: `Error: Unexpected upstream response (HTTP status code 500) when requesting ${expectedUrl}`,
         });
       });
+
+      it('should retry on failure is maximumAttempts is increased', async () => {
```

Comment:
The pedant in me can't resist... should this be 'should retry on failure **if** maximumAttempts is greater than one'?

Reply:
yep 👍 not pedantic at all!